cfme/configure/settings.py

```diff
@@ -24,27 +27,13 @@
 nav.add_branch(
     'my_settings',
     {
-        'my_settings_time_profiles':
-        [
-            lambda _: tabs.select_tab("Time Profiles"),
-            {
-                "timeprofile_new":
-                lambda _: cfg_btn('Add a new Time Profile'),
-
-                "timeprofile_edit":
-                lambda ctx: timeprofile_table.click_cell(
-                    "description", ctx.timeprofile.description),
-
-            }
-        ],
-        'my_settings_visual': [lambda _: tabs.select_tab("Visual"), {}],
-        'my_settings_default_filters': [lambda _: tabs.select_tab("Default Filters"), {}],
+
         'my_settings_default_views': [lambda _: tabs.select_tab("Default Views"), {}],
```

Comment:
Would you mind me asking if you intend to replace this force_nav branch as well in this PR ?